components/Side/ToggleMode.vue

```diff
@@ -0,0 +1,56 @@
+<script setup>
+import { ref } from "vue";
+
+const colorMode = useColorMode();
+const toggleChecked = ref(false);
+
+// Function to toggle the select dropdown
+const toggleSelect = () => {
+  colorMode.preference = toggleChecked.value ? "dark" : "light";
+};
```

Comment:
When the system theme is dark, the page will be rendered in dark mode, but the switch will show light mode.
![obraz](https://github.com/kamilziemann/nuxt_crm/assets/101877993/2e274de8-c137-4d5b-b709-6bcf0d9ee247)
You should add reading this value from local storage or You can also fix it to only dark/light